sharing/present.go

```diff
@@ -129,6 +130,11 @@ func (handler *Handler) showObject(ctx context.Context, w http.ResponseWriter, r
 		w.Header().Set("Content-Disposition", "attachment")
 	}
 	if download || !wrap {
+		contentType := mime.TypeByExtension(filepath.Ext(o.Key))
+		if contentType != "" {
+			w.Header().Set("Content-Type", contentType)
+		}
```

Comment:
`httpranger.ServeContent` already does this

Reply:
yes, I understand this. It also checks if Content-type header was set before. But I can't understand why this issue is happening https://github.com/storj/linksharing/issues/39

It is happening in prod but isn't happening in local env

Reply:
I mean, all the `.odt .docx .xlsx` file are downloaded as .zip in prod for some reason

Reply:
Yes, but since nothing has set the Content-Type so far, it muts be empty... hence the httpranger.ServeContent does the same thing.

Try clearing your mime mappings (or at least remove entries for docx & odt) alternatively -- you might be able to reproduce it by uploading a zip file with an unknown extension.

However, I think the suggestion https://github.com/storj/linksharing/issues/39#issuecomment-774162237, should be correct. If `mime.TypeByExtension` doesn't find appropriate type we should use `application/octet-stream` rather than leaving it up to the `http.DetectContentType`.